Ghidra/Features/Python/ghidra_scripts/ImportFunctionsAndLabels.py

```diff
@@ -0,0 +1,35 @@
+# Imports a file with lines in the form "symbolName 0xADDRESS function_or_label" where "f" indicates a function and "l" a label
+# @author matedealer <git@matedealer.de>
+# @category Data
+#
+
+from ghidra.program.model.symbol.SourceType import *
+import string
+
+functionManager = currentProgram.getFunctionManager()
+
+f = askFile("Give me a file to open", "Import")
+
+for line in file(f.absolutePath):  # note, cannot use open(), since that is in GhidraScript
+    pieces = line.split()
+
+    name = pieces[0]
+    address = toAddr(long(pieces[1], 16))
+    type = pieces[2]
+
+
+
+    if type == "f":
+        func = functionManager.getFunctionAt(address)
+
+        if func != None:
+            old_name = func.getName()
+            func.setName(name, USER_DEFINED)
+            print "Renamed function %s to %s at address %s" % (old_name, name, address)
```

Comment:
Change to:
print("Renamed function %s to %s at address %s" % (old_name, name, address))